pkg/controller/statusmanager/status_manager_test.go

```diff
@@ -396,26 +356,26 @@ func TestStatusManagerSetFromDaemonSets(t *testing.T) {
 
 	// Create minimal DaemonSets
 	dsA := &appsv1.DaemonSet{
-		ObjectMeta: metav1.ObjectMeta{Namespace: "one", Name: "alpha", Generation: 1},
+		ObjectMeta: metav1.ObjectMeta{Namespace: "one", Name: "alpha", Generation: 1, ClusterName: "default"},
```

Comment:
We have a bit of a problem: we consider cluster names "" and "default" to be identical. It would be good if we normalized everything, probably to "", for the purposes of status reporting.

Reply:
Will update to use `""`.

cc @kyrtapz

Reply:
Done.